### PR TITLE
chore(github): add ionitron bot to the repo 

### PR DESCRIPTION
**Pull request by rwaskiewicz**
this commit adds our internal bot 'ionitron' to the create-stencil
repository. technically, ionitron is already configured for the repo,
but needs a configuration file (this commit) to run.

the first commit adds the ionitron configuration from the stencil core
compiler repository at
https://github.com/ionic-team/stencil/blob/b2baf16ec84a944ea567e707bfcde1983e2696ac/.github/ionic-issue-bot.yml.

the next commit then tailors this config to the project

### CODE REVIEW

File: .github/ionic-issue-bot.yml [hunk 1]
```diff
@@ -11,15 +11,15 @@ closeAndLock:
         for questions about Stencil.
```
Comment (alicewriteswrongs) [inline .github/ionic-issue-bot.yml:11]: should we say 'Stencil and Stencil Store' here?

**Review by alicewriteswrongs — COMMENTED** [review]
just one small question, otherwise looks good!

File: .github/ionic-issue-bot.yml (hunk 1 above)
Comment (rwaskiewicz) [reply .github/ionic-issue-bot.yml]: Yep, I think so!

**Review by alicewriteswrongs — APPROVED** [review]
lgtm!

**Pull request merged**